### ISSUE TITLE
Project Limit Request: circt - 225 GB

### ISSUE THREAD

**Issue by mikeurbach** [pypi/support#3423]
### Project URL

https://pypi.org/project/circt

### Does this project already exist?

- [X] Yes

### New limit

225

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI

### About the project

The CIRCT project is an incubator project in the LLVM foundation that provides open source electronic design automation (EDA) compiler infrastructure. We provide Python bindings to allow users to build tooling in Python on top of the CIRCT infrastructure. You can find more info here if you're interested: https://circt.llvm.org/.

We provide wheels for different platforms on PyPI so users can easily get started with the tools without needing to compile LLVM and CIRCT from source.

### How large is each release?

~126.4 MB

Our releases include 4 Linux wheels at 18MB each, and 4 MacOS wheels at 13.6MB each.

The large size is due to the inclusion of a large native extension, compiled as part of LLVM and CIRCT. This is currently including the minimum amount of LLVM required to provide the extension, but LLVM is quite a heavy dependency.

### How frequently do you make a release?

We make weekly releases, but we also provide nightly dev releases. They are all the same size.

The nightly dev releases are the core reason I'm requesting a size increase. These make it easy for developers to test with the latest features and qualify weekly release candidates. We want to keep making nightly releases to PyPI if possible. If the size is not acceptable, we can explore other routes, like periodically deleting old nightlies, but we are generally hoping to avoid deleting old releases from PyPI if possible. We did hit the default limit this week and I just manually deleted nightly releases older than 3 months for now.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by mikeurbach** [comment]
Any update on whether we can increase the project limit here? We have once again hit the 10GB limit, and I have bought a little more time by manually removing old nightly dev releases. If there is no update, we will reduce our nightly releases in the meantime.

**Comment by di** [comment]
It's a little hard for me to get more details here because the current releases don't have project descriptions or any project links. Is this the source repository for this project? https://github.com/llvm/circt

**Comment by mikeurbach** [comment]
Sorry about that @di , yes that is the project and we should provide a description. There is also more information about the overall CIRCT project at [circt.llvm.org](https://circt.llvm.org/). Please let me know if you have any other questions I can help with, since we're lacking a good description on the project.

**Comment by di** [comment]
Hey @mikeurbach, after reviewing again I'm happy to unblock you but a little confused what you actually need here. The title says "225 GB" but I think you're actually asking for a 225MB individual file size limit (up from the default of 100MB)? Or are you actually asking to go from the 10GB project size limit to 225GB? Thanks!

**Comment by mikeurbach** [comment]
Oh sorry it wasn't clear when I filled out the description. We are ok in terms of the individual file size limit, I was asking to increase the 10GB project size limit to 225GB. That is what I calculated to give us 5 years of headroom at the current growth rate. I'm not sure how the provisioning works, so if 225GB is too much, but some other limit would be more feasible, any increase over the 10GB limit would help!
